lib/github/github/reviews.ex

```diff
@@ -8,16 +8,30 @@ defmodule BorsNG.GitHub.Reviews do
 
   @spec from_json!([map]) :: t
   def from_json!(json) do
-    json
+    reviews = json
     # Count only the latest review from a user,
     # by deduplicating using the user id
     |> Enum.reduce(%{}, fn
       %{"state" => "COMMENTED"}, acc ->
         acc
-      %{"user" => %{"id" => uid}, "state" => state}, acc ->
+      %{"user" => %{"login" => uid}, "state" => state}, acc ->
         Map.update(acc, uid, state, fn _ -> state end)
       end)
+
+    # Get the list of users who have approved this PR
+    approvers = reviews
+                |> Enum.flat_map(fn {_uid, state} ->
+      case state do
+        # Ignore dismissed reviews
+        "DISMISSED" -> []
+        # Pass just the username onward
+        id -> [_uid]
```

Comment:
if you are using the variable, if possible, do not prepend it with an underscore `_`, this in elixir terms represents a binding/variable that you will not be using, and this convention makes some analysis ignore the binding

Reply:
Gotcha.  ⚡️